src/components/06_wrappers/ErrorBoundary/ErrorBoyndary.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
filename has a typo here

Reply:
I fixed that in https://github.com/jsdrupal/drupal-admin-ui/pull/73